mteb/tasks/Clustering/deu/BlurbsClusteringP2P.py

```diff
@@ -33,3 +34,48 @@ class BlurbsClusteringP2P(AbsTaskClustering):
         n_samples={"test": 174637},
         avg_character_length={"test": 664.09},
     )
+
+
+class BlurbsClusteringP2PFast(AbsTaskClustering):
```

Comment:
I was wondering why first Fast tasks do not inherit from `AbsTaskClusteringFast` ?

Reply:
This is just a downsampled version of BlurbsClusteringP2P (see dataset_transform). The labels in the different clusters don't seem to be consistent so I could not use the approach in Fast (also see comment below class def). It might be possible to now redo this dataset using the hierarchical clustering (@x-tabdeveloping) which was not present at the time.

Reply:
(feel free to create an issue on this)

Reply:
Okay, we'll keep it like this here and open an issue for the hierarchical clustering, thanks!